cmd/test-utils_test.go

```diff
@@ -1949,6 +1949,9 @@ func registerBucketLevelFunc(bucket *router.Router, api objectAPIHandlers, apiFu
 			// Register PostPolicy handler.
 			bucket.Methods("POST").HeadersRegexp("Content-Type", "multipart/form-data*").HandlerFunc(api.PostPolicyBucketHandler)
 			// Register GetObject handler.
+		case "HeadObject":
+			bucket.Methods("Head").Path("/{object:.+}").HandlerFunc(api.HeadObjectHandler)
+			// Register PutObject handler.
```

Comment:
Comment should be "Register GetObject handler"?
Also line 1951 should reference HeadObject (instead of GetObject).